options.go

```diff
@@ -266,6 +282,26 @@ func (opts *Options) Initialize() error {
 		}
 	}
 
+	if opts.SystemMetrics.Endpoint.Host == "" {
+		opts.SystemMetrics.Endpoint.Host = DefaultSystemMetricsHost
```

Comment:
@codeboten did we consider making this default to the same endpoint as the collector? That would simplify the config even more for our internal services